docs/docs/dev_docs/tutorials/token_portal/setup.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Setup and Installation
+---
+
+In this step, we’re going to
+
+1. Install prerequisites
+2. Create a yarn project to house everything
+3. Create a nargo project for our Aztec contract
+4. Create a hardhat project for our Ethereum contract(s)
+5. Import all the Ethereum contracts we need
+6. Create a yarn project for our TS
+
+We recommend going through this setup to fully understand where things live.
+
+However if you’d rather skip this part, our dev-rels repo contains the starter code here.
+
+# Prerequisites
+
+- [node v18+](https://github.com/tj/n)
+- [docker](https://docs.docker.com/)
+- [Aztec sandbox](https://docs.aztec.network/dev_docs/getting_started/sandbox)
```

Comment:
```suggestion
- [A running aztec sandbox](https://docs.aztec.network/dev_docs/getting_started/sandbox)
```

Reply:
You could also delete node, docker dependencies since they are covered in the aztec sandbox getting started section